.github/workflows/testNginxForAzureDeploy.yml

```diff
@@ -52,33 +52,30 @@ jobs:
         shell: bash
         run: |
           sed -i 's/000000/'"$GITHUB_RUN_ID"'/g' test/configs/multi/nginx.conf
-          cat test/configs/single/nginx.conf
+          cat test/configs/multi/nginx.conf
           sed -i 's/000000/'"$GITHUB_RUN_ID"'/g' test/configs/multi/conf.d/proxy.conf
           cat test/configs/multi/conf.d/proxy.conf
       - name: "Sync NGINX configuration and certificate to NGINX for Azure - multi file"
-        uses: nginxinc/nginx-for-azure-deploy-action@v0.2.0
+        uses: nginxinc/nginx-for-azure-deploy-action@v0.2.1
         with:
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
           resource-group-name: $TEST_RESOURCE_GROUP_NAME
           nginx-deployment-name: $NGINX_DEPLOYMENT_NAME
-          nginx-resource-location: "westcentralus"
+          nginx-deployment-location: "westcentralus"
```

Comment:
Maybe this was "resource" to avoid confusion between an "NGINX Deployment" and an "Azure Deployment"?

Reply:
I guess? Oh well.